docker/reqmgr2ms-unmerged/run.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
```

Comment:
the same argument here, may be you can replace bash with sh

Reply:
And here already I am 100% sure that I not only need bash, but also that If I could have a specific version of bash, more than half of the variable expansion hacking in the `run.sh` which took me hours of debugging, would not have happened at all :)